scenarios/kubernetes_verify.py

```diff
@@ -79,9 +79,10 @@ def get_git_cache(k8s):
 
 def branch_to_tag(branch):
     verify_branch = re.match(r'release-(\d+\.\d+)', branch)
-    if not verify_branch:
-        return VERSION_TAG['default']
-    return VERSION_TAG[verify_branch.group(1)]
+    key = 'default'
```

Comment:
`key = (verify_branch and verify_branch.group(1)) or 'default'` 😈

Reply:
people seem to not like these expressions, per previous reviews, IIRC :(
personally I'm a fan of them :upside_down_face: